### PR TITLE
feat(sip-connector): add participant events listeners

### PR DESCRIPTION
**Pull request by papamishka911**
notify about current user has added or removed from moderators list

### CODE REVIEW

File: src/SipConnector.ts [hunk 1]
```diff
@@ -1083,6 +1091,38 @@ export default class SipConnector {
 
       this._maybeTriggerChannelsNotify(channelsInfo);
     }
+
+    if (header.cmd === CMD_ADDED_TO_LIST_MODERATORS) {
```
Comment (Krivega) [inline src/SipConnector.ts:1095]: use swith

File: src/SipConnector.ts [hunk 2]
```diff
@@ -1083,6 +1091,38 @@ export default class SipConnector {
...
+      this._sessionEvents.trigger('participant:removed-from-list-moderators', headersParametersModeratorsList);
+    }
+  };
+
+  _maybeTriggerAddedToListModeratorsNotify = ({ conference }: TAddedToListModeratorsInfoNotify) => {
+    const headersParametersModeratorsList: TParametersModeratorsList = {
+      conference,
+    };
+
+    if (this.session) {
```
Comment (Krivega) [inline src/SipConnector.ts:1123]: why? remove

File: src/SipConnector.ts [hunk 3]
```diff
@@ -1083,6 +1091,38 @@ export default class SipConnector {
...
+      this._maybeTriggerRemovedFromListModeratorsNotify(addedToListModeratorsInfo);
+    }
+  };
+
+  _maybeTriggerRemovedFromListModeratorsNotify = ({ conference }: TAddedToListModeratorsInfoNotify) => {
+    const headersParametersModeratorsList: TParametersModeratorsList = {
+      conference,
+    };
+
+    if (this.session) {
```
Comment (Krivega) [inline src/SipConnector.ts:1113]: remove

**Comment by Krivega** [comment]
how about tests?

**Review by Krivega — CHANGES_REQUESTED** [review]

File: src/SipConnector.ts (hunk 3 above)
Comment (papamishka911) [reply src/SipConnector.ts]: removed

File: src/SipConnector.ts (hunk 2 above)
Comment (papamishka911) [reply src/SipConnector.ts]: removed

File: src/SipConnector.ts (hunk 1 above)
Comment (papamishka911) [reply src/SipConnector.ts]: added

**Comment by papamishka911** [comment]
> how about tests?

module tests has added

**Pull request merged**